Explorer/Assets/Scripts/Utility/ParcelMathHelper.cs

```diff
@@ -149,8 +149,10 @@ public static bool Contains(this in SceneCircumscribedPlanes boundingPlanes, Bou
             Vector3 min = bounds.min;
             Vector3 max = bounds.max;
 
-            return boundingPlanes.MinX < min.x && boundingPlanes.MaxX > max.x
-                                               && boundingPlanes.MinZ < min.z && boundingPlanes.MaxZ > max.z;
+            return Mathf.Ceil(boundingPlanes.MinX) <= Mathf.Ceil(min.x) &&
+                   Mathf.Ceil(boundingPlanes.MaxX) >= Mathf.Ceil(max.x) &&
+                   Mathf.Ceil(boundingPlanes.MinZ) <= Mathf.Ceil(min.z) &&
+                   Mathf.Ceil(boundingPlanes.MaxZ) >= Mathf.Ceil(max.z);
```

Comment:
Strange solution. Why not just `bounds.Expand(Vector3.kEpsilon)` before the comparison?

Reply:
Or Expend by `-Vector3.kEpsilon`.

Reply:
The difference in this case is about `0,03223`, but I think `Vector3.kEpsilon` is much less than that, right?

![image](https://github.com/user-attachments/assets/ada11922-9435-4ffe-9837-554f801efbdc)

Reply:
@AnsisMalins I finally decided to make use of the `bounds.Expand(...)` with a `BOUNDS_OFFSET_EPSILON` of `0.1f` 👍 I think this makes it much clearer and simpler. Thanks for the feedback!

Changes done [here](https://github.com/decentraland/unity-explorer/pull/3400/commits/9f967237195a598f2b15b783620e03e8a4064725).